src/adcm_pytest_plugin/steps/actions.py

```diff
@@ -221,7 +221,7 @@ def _run_action_and_assert_result(
             task = obj.action(name=action_name).run(**kwargs)
         except ErrorMessage as err:
             ADCMError.raise_if_suitable(err.error.title)
-            raise ErrorMessage from err
+            raise ErrorMessage(err.error.title) from err
```

Comment:
probably you can just `raise` without a new exception

Reply:
Changed